### ISSUE TITLE
Missing "Gamma" symbol

### ISSUE THREAD

**Issue by rafvrab** [learn-co-curriculum/dsc-gradient-boosting-and-weak-learners#1]
## Link to Canvas

https://learning.flatironschool.com/courses/5876/pages/gradient-boosting-and-weak-learners?module_item_id=508733

## Issue Subtype

Section: Learning rates
Sentence "You'll often see learning rates denoted by the symbol, -- this is the greek letter, gamma. Don't worry if you're still hazy on the concept of gradient descent -- we'll explore it in much more detail when we start studying deep learning!"
Symbol for the Gamma letter is missing.

- [ ] Master branch code
- [ ] Solution branch code
- [ ] Code tests
- [ ] Layout/rendering issue
- [ ] Instructions unclear
- [ ] Other (explain below)

### Source

```

```

## What OS Are You Using?

- [ ] OS X
- [ ] Windows
- [ ] WSL
- [ ] Linux
- [ ] Saturn Cloud from Canvas

**Comment by bpurdy-ds** [comment]
Thanks for bringing this to our attention. We'll get it fixed.

**Comment by bpurdy-ds** [comment]
Fixed.
